crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -413,7 +413,7 @@ pub struct SuiTBlsSignRandomnessObjectResponse {
 #[allow(clippy::large_enum_variant)]
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
 pub struct SuiExecuteTransactionResponse {
-    pub certificate: SuiCertifiedTransaction,
+    pub certificate: Option<SuiCertifiedTransaction>,
```

Comment:
nit: it will be great to add a comment about under which condition this can be None

Reply:
Could you also change the TS definition in https://github.com/MystenLabs/sui/blob/main/sdk/typescript/src/types/transactions.ts#L241 from `certificate: CertifiedTransaction,` to `certificate: optional(CertifiedTransaction),`?

Reply:
I thought I did? Or is there another place I need to change?

Reply:
Oops sorry for missing the change